services/director-v2/tests/unit/test_modules_dynamic_sidecar_docker_service_specs_volume_remover.py

```diff
@@ -1,10 +1,205 @@
 # pylint: disable=redefined-outer-name
+
+from pathlib import Path
+from typing import AsyncIterator
+
+import aiodocker
 import pytest
+from aiodocker.volumes import DockerVolume
+from faker import Faker
 from pydantic import parse_obj_as
 from simcore_service_director_v2.modules.dynamic_sidecar.docker_service_specs.volume_remover import (
+    SH_SCRIPT_REMOVE_VOLUMES,
     DockerVersion,
 )
 
+# UTILS
+
+
+def _get_source(run_id: str, node_uuid: str, volume_path: Path) -> str:
+    reversed_path = f"{volume_path}"[::-1].replace("/", "_")
+    return f"dyv_{run_id}_{node_uuid}_{reversed_path}"
+
+
+async def run_command(
+    docker_version: DockerVersion, volume_names: list[str]
+) -> list[str]:
+    async with aiodocker.Docker() as docker_client:
+        volume_names_seq = " ".join(volume_names)
+        formatted_command = SH_SCRIPT_REMOVE_VOLUMES.format(
+            volume_names_seq=volume_names_seq, retries=5, sleep=1
+        )
+        print("Container will run:\n%s", formatted_command)
+        command = ["sh", "-c", formatted_command]
+
+        container = await docker_client.containers.run(
+            config={
+                "Cmd": command,
+                "Image": f"docker:{docker_version}-dind",
+                "HostConfig": {"Binds": ["/var/run/docker.sock:/var/run/docker.sock"]},
+            },
+        )
+        await container.start()
+        await container.wait()
+
+        logs = await container.log(stderr=True, stdout=True)
+
+        await container.delete(force=True)
+
+        return "".join(logs)
+
+
+# FIXTURES
+
+
+@pytest.fixture
+def swarm_stack_name() -> str:
+    return "test_stack"
+
+
+@pytest.fixture
+def study_id(faker: Faker) -> str:
+    return faker.uuid4()
+
+
+@pytest.fixture
+def node_uuid(faker: Faker) -> str:
+    return faker.uuid4()
+
+
+@pytest.fixture
+def run_id(faker: Faker) -> str:
+    return faker.uuid4()
+
```

Comment:
I think we should put all these simple fixtures in common. Perhaps somewhere in pytest_simcore?

![image](https://user-images.githubusercontent.com/32402063/195157295-c7136daf-f6e5-4012-ba65-d0ee516cd281.png)

Reply:
We need to distinguish between flavours. Sometimes we require a `str` and sometimes the `UUID/NodeId` version of the fixture. here I have them as strings as I did not want to bother with casting to string again.

Not an easy feat, but yes we'd like to have a clear module where all these project related values are generated in different flavours. We'd need to agree that those will be used from now on and start refactoring tests as we go.